public/controllers/register-agent/containers/steps/steps.tsx

```diff
@@ -0,0 +1,244 @@
+import React, { Fragment, useState } from 'react';
+import {
+  EuiSteps,
+  EuiText,
+  EuiTitle,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiCallOut,
+  EuiPopover,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+import { InputForm } from '../../../../components/common/form';
+import './steps.scss';
+import {
+  REGISTER_AGENT_DATA_STEP_THREE,
+  REGISTER_AGENT_DATA_STEP_TWO,
+} from '../../utils/register-agent-data';
+import { webDocumentationLink } from '../../../../../common/services/web_documentation';
+import { PLUGIN_VERSION_SHORT } from '../../../../../common/constants';
+
+const popoverServerAddress = (
+  <span>
+    Learn about{' '}
+    <a
+      href={webDocumentationLink(
+        'user-manual/reference/ossec-conf/client.html#groups',
+        PLUGIN_VERSION_SHORT,
+      )}
+      target='_blank'
+      rel='noopener noreferrer'
+    >
+      Server address.
+    </a>
+  </span>
+);
+
+const popoverAgentName = (
+  <span>
+    Learn about{' '}
+    <a
+      href={webDocumentationLink(
+        'user-manual/reference/ossec-conf/client.html#enrollment-agent-name',
+        PLUGIN_VERSION_SHORT,
+      )}
+      target='_blank'
+      rel='noopener noreferrer'
+    >
+      Assigning an agent name.
+    </a>
+  </span>
+);
+
+export const Steps = ({
+  needsPassword,
+  hidePasswordInput,
+  agentGroup,
+  form,
+  osCard,
+}) => {
+  const warningForAgentName =
+    'The agent name must be unique. It can’t be changed once the agent has been enrolled.';
+
+  const [isPopoverServerAddress, setIsPopoverServerAddress] = useState(false);
+  const [isPopoverAgentName, setIsPopoverAgentName] = useState(false);
+
+  const onButtonServerAddress = () =>
+    setIsPopoverServerAddress(
+      isPopoverServerAddress => !isPopoverServerAddress,
+    );
+  const closeServerAddress = () => setIsPopoverServerAddress(false);
+
+  const onButtonAgentName = () =>
+    setIsPopoverAgentName(isPopoverAgentName => !isPopoverAgentName);
+  const closeAgentName = () => setIsPopoverAgentName(false);
+
+  const firstSetOfSteps = [
+    {
+      title: (
+        <EuiTitle className='stepTitle'>
+          <p>Select the package to download and install on your system:</p>
+        </EuiTitle>
+      ),
+      children: osCard,
+      status: form.fields.operatingSystemSelection.value
+        ? 'complete'
+        : 'current',
+    },
+    {
+      title: (
+        <EuiFlexGroup>
+          {/* <EuiFlexItem className='stepTitle'>
+            <p>Server address</p>
+            <EuiIconTip content={tooltipContent} position='right' />
+          </EuiFlexItem> */}
```

Comment:
**nitpick**: remove it

Reply:
Removed in commit [777d15d](https://github.com/wazuh/wazuh-kibana-app/pull/5554/commits/777d15df38e5e1f1d42b05b49a9cdd8f3ccbf29d)